src/libertem/executor/pipelined.py

```diff
@@ -898,13 +916,14 @@ def close(self):
                     try:
                         with worker_info.queues.response.get(block=False) as msg:
                             logger.warning(f"got message on close: {msg[0]}")
-                    except WorkerQueueEmpty:
+                    except (WorkerQueueEmpty, TypeError):
```

Comment:
This is curious - why do we need to catch `TypeError` here?

Reply:
I think this is a leftover from when I was trying to debug #1339   , I will remove it.